### ISSUE TITLE
Mobile | Création compte

### ISSUE THREAD

**Issue by Thornolf** [recipyhq/recipy#344]
Impossible de créer un compte sur Mobile, avec les adresses Gmail et Wanadoo, les emails de confirmations n'arrivent jamais ni mêem dans les pourriels.

**Comment by guillobits** [comment]
Ça doit être général au web aussi vu que c'est le web qui send me mail. Peut être du au fait qu'on ai force recipy.fr ? 
Analyser les logs

**Comment by Thornolf** [comment]
Non, les gens ont pu créer directement un compte via le site et chose étonnante avec la même adresse email déjà utilisé lors de la tentative sous mobile.
